tests/unit/test_routes.py

```diff
@@ -32,6 +34,19 @@ def test_get_resources(module_client, module_db):
         assert (resource.get('category'))
         assert (isinstance(resource.get('languages'), list))
 
+    ua = datetime.now() + timedelta(days=-7)
+    uaString = ua.strftime('%m-%d-%Y')
+    response = client.get(f"/api/v1/resources?updated_after={uaString}")
+    assert (response.status_code == 200)
+
+    # If updated_after date is after today, then should return a 422
+    ua = datetime.now() + timedelta(days=1)
+    uaString = ua.strftime('%m-%d-%Y')
+    response = client.get(f"/api/v1/resources?updated_after={uaString}")
+    assert (response.status_code == 422)
+    assert (response.json.get('errors')[0].get('code') == "unprocessable-entity")
```

Comment:
split this out into separate methods.

I'd suggest splitting this out into named methods
`test_get_resources_post_date_failure` 

Each test should ideally minimize the number of assertions so when one fails we can tell why.